File: csharp/bypstest-ser/src-ser/com/wilutions/byps/test/api/inherit/BioLemonService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using com.wilutions.byps;

namespace com.wilutions.byps.test.api.inherit
{
	
	// checkpoint com.wilutions.byps.gen.cs.GenRemoteClass:93
	public interface BioLemonService : BRemote, 
		BioFruitService, 
		LemonService {
	
		int UseParing();
		void UseParing(BAsyncResult<int> asyncResult) ;
		Task<int> UseParingAsync();
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/inherit/BioLemonService.java
package byps.test.api.inherit;

import byps.RemoteException;

/**
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 */
public interface BioLemonService extends BioFruitService, LemonService {

  int useParing() throws RemoteException;
}
